devops/resources/create-data-czar.sh

```diff
@@ -9,17 +9,18 @@ create_virtualenv --python=python3.8 --clear
 set -u
 
 cd "$WORKSPACE/configuration"
-pip install -r requirements.txt
+pip install -r util/jenkins/requirements.txt
 . util/jenkins/assume-role.sh
 
 assume-role ${ROLE_ARN}
 
 cd util/create_data_czar
 
 # Create Policy
-if CREATE_ORG == "true"  then;
+if [ ${CREATE_ORG} == "true"  ]; then
```

Comment:
This if statement makes me wonder if we should have a dry run flag that can be set to make the job print what it would do without actually doing it.  Would that be useful?